### DISCUSSION TITLE
Date picker Calendar Format

### DISCUSSION THREAD

**Discussion by Betacord360** [iSoron/uhabits#1237] [Feature Requests]
Can the date picker Calendar be in a regular format like any month wall calendar ever? 

My view in the app:
![Screenshot_20211212-013819_Habits](https://user-images.githubusercontent.com/96001685/145707440-435b8f91-0d73-4e9f-bd2b-067d6a08e3f7.jpg)

Normal Calendar Format:
![lt02809252_quantized](https://user-images.githubusercontent.com/96001685/145707445-69e06345-5cf8-4f86-b45d-6d50a27d4c87.png)

**Comment by hiqua** [comment]
One problem with your format is that we don't see past months.

**Comment by davi-jorge-art** [comment]
> Can the date picker Calendar be in a regular format like any month wall calendar ever?
> 
> My view in the app: ![Screenshot_20211212-013819_Habits](https://user-images.githubusercontent.com/96001685/145707440-435b8f91-0d73-4e9f-bd2b-067d6a08e3f7.jpg)
> 
> Normal Calendar Format: ![lt02809252_quantized](https://user-images.githubusercontent.com/96001685/145707445-69e06345-5cf8-4f86-b45d-6d50a27d4c87.png)

Maybe it will be "solved" with https://github.com/iSoron/uhabits/discussions/2043#discussion-7144058
@hiqua
